src/encoda.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Listener,
+  // TODO: Now that Executa re-exports logga
+  // and schema, shall we use those instead of
+  // potentially having version conflicts
```

Comment:
I think if we have a compatible version range defined, it leaves room removing `logger` or `schema` changes to Executa down the road. But I'm not set on this either way.